index.html

```diff
@@ -354,7 +354,7 @@ <h3>StatusList2021Entry</h3>
               <td>statusListIndex</td>
               <td>
 The <code>statusListIndex</code> property MUST be an arbitrary size integer
-greater than or equal to 0, expressed as a string. The value identifies the bit
+greater than or equal to 0, expressed as a string. The value identifies the
 position of the status of the <a>verifiable credential</a>.
```

Comment:
```suggestion
position of the status of the <a>verifiable credential</a>.
```
Is this position based on `0` or `1`?

Reply:
Wouldn't that depend on what language you are implementing in? E.g. for most 0 indicates first position, but there are exceptions

Reply:
It might. I think we should say so, either way.

Reply:
really depends on your programming language - in most cases `0` indicates the first position, but there are exceptions of course.  If we wanted to pick one here, I would say `0` (and I believe that is inline with the existing spec, though it is not called out, but I would have to check)